### DISCUSSION TITLE
proton treatment machine SAD

### DISCUSSION THREAD

**Discussion by du-guyue** [e0404/matRad#672] [Q&A] [answered]
Hello everyone, 
While running matRad_example5_protons, I noticed that machine.meta.SAD = 10000, which is a parameter used in generateStf, calcGeoDists etc. However, I am aware that clinical spot-scanning proton therapy treatment machines typically have two deflecting magnets, responsible for X and Y direction deflections, resulting in two SAD parameters (SADx and SADy) in the machine. My question is whether matRad currently only supports machine models with one SAD parameter, and all the source points of rays within a beam are considered the same?

Thank you,
Yue

**Comment by wahln** [comment]
In the current release, the pencil-beam algorithm assumes radially symmetric kernels (since this was the case for the clinical algorithm we validated it against). This is usually good enough since most machines create approximately radially symmetric spots at isocenter. The SAD thus defines a virtual source and is not representing one of the scanning magnets itself. All rays within a beam then have the same virtual source point -SAD away from the isocenter

On the branch becoming the next release, we have implemented Monte Carlo interfaces to MCsquare and TOPAS that use phase space parameterizations at the nozzle exit, and differentiate between x and y geometry (spread, divergence, correlation in each of the two directions).

We also have experimental implementations where the pencil-beam algorithm can use different kernels in X and Y and work with asymmetric spot sizes in air during dose calculation. However, the ray geometry is still approximated with the virtual source (usually in the middle between x & y scanning magnet).

**Reply by du-guyue** [reply]
Thank you for your reply! :)

I agree with you that the clinical spots are roughly symmetrical. However, I'm still confused about whether all the rays (within a beam) deflected by the X and Y magnets can be back-projected to the "same" virtual source point. Is this "sameness" an approximation? In other words, are these rays' virtual source points close enough to be considered the same, but not exactly coincident?

**Reply by wahln** [reply, marked as answer]
Yes, this approximation is considered "good enough" in many cases since the SAD is big enough such that you have only minimal deviations in the incident X/Y angles from the virtual source angle (example realistic values SAD = 2m, SAD_x = 1850, SAD_y 2150). The difference is there, but it is small compared to other approximation errors.

**Reply by du-guyue** [reply]
OK, I see. Thank you very much! :)
